iina/FilterWindowController.swift

```diff
@@ -164,19 +164,34 @@ class FilterWindowController: NSWindowController, NSWindowDelegate {
     saveFilter(filters[row])
   }
 
+  /// User activates or deactivates previously saved audio or video filter
+  /// - Parameter sender: A checkbox in lower portion of filter window
   @IBAction func toggleSavedFilterAction(_ sender: NSButton) {
     let row = savedFiltersTableView.row(for: sender)
-    let filter = savedFilters[row]
+    let savedFilter = savedFilters[row]
     let pc = PlayerCore.active
-    if sender.state == .on {
-      if pc.addVideoFilter(MPVFilter(rawString: filter.filterString)!) {
-        pc.sendOSD(.addFilter(filter.name))
-      }
+
+    // choose approriate add/remove functions for .af/.vf
+    var addFilterFunction: (MPVFilter) -> Bool
+    var removeFilterFunction: (MPVFilter) -> Bool
+    if filterType == MPVProperty.vf {
+      addFilterFunction = pc.addVideoFilter
+      removeFilterFunction = pc.removeVideoFilter
     } else {
-      if pc.removeVideoFilter(MPVFilter(rawString: filter.filterString)!) {
+      addFilterFunction = pc.addAudioFilter(_:)
```

Comment:
Be consistent with other functions references, remove the parameter list

Reply:
You mean it should be "addFilterFunction = pc.addAudioFilter", right?  You've merged/closed, so it's just an FYI / "for future reference", right?  Thanks for the feedback.  Should I hit Resolve Conversation?  [sorry, i'm a noob]

Reply:
Yes. There’s nothing you need to do; just keep it in mind for the future.